jupyter/trustyai/ubi8-python-3.8/test/test_notebooks.ipynb

```diff
@@ -0,0 +1,88 @@
+    "\n",
+    "    def test_python_version(self): \n",
+    "        expected_major_minor = '3.8' # Set the expected version (x.y) \n",
+    "        actual_major_minor = '.'.join(python_version().split('.')[:2])\n",
+    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
+    "\n",
+    "    def test_trustyai_version(self): \n",
+    "        expected_major_minor = '0.2' # Set the expected version (x.y) \\n\n",
+    "        actual_major_minor = '.'.join(trustyai.__version__.split('.')[:2])\\n \n",
+    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
+    "    \n",
+    "    def test_fairnessmetrics(self):\n",
+    "        url_unbiased = \"https://raw.githubusercontent.com/dibryant/notebooks/trustyai/jupyter/trustyai/ubi9-python-3.9/test/income-unbiased.csv\"\n",
+    "        nobias = pd.read_csv(url_unbiased, index_col=False)\n",
+    "        \n",
+    "        nobias = pd.read_csv(url_unbiased, index_col=False)\n",
+    "        nobias.groupby(['gender', 'income'])['income'].count()\n",
+    "        nobias.groupby(['gender', 'income'])['income'].count().unstack().plot.bar()\n",
+    "        \n",
+    "        nobias_privileged = nobias[nobias.gender == 1]\n",
+    "        nobias_unprivileged = nobias[nobias.gender == 0]\n",
+    "        favorable = output(\"income\", dtype=\"number\", value=1)\n",
+    "        score = statistical_parity_difference(privileged=nobias_privileged,\n",
+    "                                              unprivileged=nobias_unprivileged,\n",
+    "                                              favorable=[favorable])\n",
+    "        print(score)\n",
+    "    def test_datafairness(self):\n",
+    "        url_biased = \"https://raw.githubusercontent.com/dibryant/notebooks/trustyai/jupyter/trustyai/ubi9-python-3.9/test/income-biased.csv\"\n",
+    "        bias = pd.read_csv(url_biased, index_col=False)\n",
+    "        \n",
+    "        bias = pd.read_csv(url_biased, index_col=False)\n",
+    "        bias.groupby(['gender', 'income'])['income'].count()\n",
+    "        bias.groupby(['gender', 'income'])['income'].count().unstack().plot.bar()\n",
+    "        \n",
+    "        bias_privileged = bias[bias.gender == 1]\n",
+    "        bias_unprivileged = bias[bias.gender == 0]\n",
+    "        favorable = output(\"income\", dtype=\"number\", value=1)\n",
+    "        score = statistical_parity_difference(privileged=bias_privileged,\n",
+    "                                              unprivileged=bias_unprivileged,\n",
+    "                                              favorable=[favorable])\n",
+    "        print(score)\n",
+    "\n",
+    "suite = unittest.TestLoader().loadTestsFromTestCase(TestTrustyaiNotebook)\n",
+    "unittest.TextTestRunner().run(suite)\n",
+    "\n"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3.9",
```

Comment:
This notebook is for ubi8 flavor, so it needs to be changed to 3.8.
To do that you need to run the contents of the notebook on a ubi8 flavor (In order to be corrected automatically)